TeachingRecordSystem/tests/TeachingRecordSystem.Api.UnitTests/DummyTest.cs

```diff
@@ -0,0 +1,8 @@
+namespace TeachingRecordSystem.Api.UnitTests;
+
+public class DummyTest
+{
+    // Dummy test so that CI passes
+    [Fact]
+    public void AlwaysPasses() { }
```

Comment:
This will be removed when we have tests added but we need at least one to make CI happy.